WhaleSpotting/react-app/src/components/FetchWeatherData.tsx

```diff
@@ -1,5 +1,6 @@
+/*eslint-disable*/
```

Comment:
Do we still need the changes to this file if we have moved all the admin changes?

Reply:
removed the file